### PR TITLE
Project Metrics: Add a display page for project metrics

### PR DESCRIPTION
**Pull request by rcebulko**
App currently live at https://amp-project-metrics.appspot.com/

### CODE REVIEW

File: project-metrics/metrics_service/templates/show_metrics.html [hunk 1]
```diff
@@ -0,0 +1,122 @@
+{% from 'metric_guage.html' import metric_guage %}
+
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Project Metrics: {{ github_repo }}</title>
+
+  <style type="text/css">
```
Comment (rcebulko) [inline project-metrics/metrics_service/templates/show_metrics.html]: Yes, this is ugly. But serving static files in flask is a whole thing so, until we know what we're actually doing with this page, embedding it here seems reasonable.

Comment (danielrozenberg) [reply project-metrics/metrics_service/templates/show_metrics.html]: This is a standard for AMP pages. In fact, have you considered making this page an AMP page? ;)

File: project-metrics/metrics_service/templates/metric_guage.html [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{% macro metric_guage(metric) -%}
+<div class='metric-guage metric-score-{{ metric.score.name.lower() }}'>
```
Comment (danielrozenberg) [inline project-metrics/metrics_service/templates/metric_guage.html]: Use double-quotes (") to delimit HTML attributes (here and in the other HTML file)

File: project-metrics/metrics_service/templates/show_metrics.html [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+    .metric-gauge-arc-background {
+      z-index: 1;
+      position: absolute;
+      background-color: #EEEEEE;
+      width: 200px;
+      height: 100px;
+      top: 100px;
+      margin-left: auto;
+      margin-right: auto;
+      border-radius: 0px 0px 100px 100px;
```
Comment (danielrozenberg) [inline project-metrics/metrics_service/templates/show_metrics.html]: No `px` after 0

File: project-metrics/metrics_service/templates/show_metrics.html (hunk 1 above)
Comment (rcebulko) [reply project-metrics/metrics_service/templates/show_metrics.html]: Shhhh...

File: project-metrics/metrics_service/templates/metric_guage.html (hunk 2 above)
Comment (rcebulko) [reply project-metrics/metrics_service/templates/metric_guage.html]: Done

File: project-metrics/metrics_service/templates/show_metrics.html (hunk 3 above)
Comment (rcebulko) [reply project-metrics/metrics_service/templates/show_metrics.html]: Done

File: project-metrics/metrics_service/templates/show_metrics.html (hunk 1 above)
Comment (rcebulko) [reply project-metrics/metrics_service/templates/show_metrics.html]: I can do that in a later PR if you think it's necessary :stuck_out_tongue:

**Pull request merged**